src/test/java/tests/tutby/Tests.java

```diff
@@ -0,0 +1,33 @@
+package tests.tutby;
+
+import driver.Config;
+import driver.DriverFactory;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import web_pages.TutByHomePage;
+
+public class Tests{
+    static WebDriver driver;
+    TutByHomePage tutByHomePage;
+
+    @BeforeAll
+    public static void initDriver(){
```

Comment:
the same!, static fields = evil!

Reply:
The method with BeforeAll annotation must be static according to documentation JUnit5

Reply:
There no need to use static even in BeforeAll